README.md

```diff
@@ -3,8 +3,6 @@
 # What is MixedRealityToolkit-vNext
 The V1 Mixed Reality Toolkit is a collection of scripts and components intended to accelerate development of applications targeting Microsoft HoloLens and Windows Mixed Reality headsets.
 
-> ### For details on the different branches used in the Mixed Reality Toolkit repositories, check this [Branch Guide here](/BranchGuide.md).
```

Comment:
Should we leave this link and replace it with the wiki version?

Reply:
we could.

Reply:
done.